packages/link/postcss.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  plugins: [
+    require('autoprefixer'),
+    require('postcss-nesting'),
+    require('postcss-custom-properties')({
+      importFrom: [
```

Comment:
Is importFrom strictly necessary?  

I don't understand all the constraints of this solution, so I'm not sure. But conceptutally, I'm thinking it's not, based on:

1. Components all have psds-normalize as a peerDep
2. psds-normalize puts all core vars in :root

So, the vars should simply be usable (even without import), because they're in scope at runtime.

Reply:
yep it's necessary to get those css vars during compilation.